netlify.toml

```diff
@@ -4,4 +4,8 @@
 [build]
   base = "www"
   publish = "_site"
-  command = "npx pnpm i --store=node_modules/.pnpm-store && npm run build"
+  command = """\
+    npx pnpm i --filter slinkity --store=node_modules/.pnpm-store &&\
+    npx pnpm i --store=node_modules/.pnpm-store &&\
+    npm run build\
+    """
```

Comment:
hey @Holben888 Should we use the development version for the docs?

Reply:
I don't see why not! this will use only use the development / canary version when on the `main-canary-0.6.0` branch. This means we _won't_ accidentally deploy our docs to `slinkity.dev` using a canary version of Slinkity. Also lets us dogfood the canary on our docs for deploy previews (as we've done here). Curious if there's any problems you could see though

Reply:
aa ok, I hadn't seen the target branch. Yes.i think is okay.

hier you can also write: 
```
npx pnpm i --filter slinkity --filter slinkity-docs --store=node_modules/.pnpm-store ...
```
why slinkity-docs, well that is the name in package.json :D

Reply:
Ah HA, didn't realize you could pass multiple filters! This is much nicer. Will change 👍

Reply:
Ah shoot, this actually causes the same problem as before 😢 

Here's the bug I was hitting:
1. `pnpm` installs all necessary dependencies
2. While installing the doc dependencies, it discovers the Slinkity CLI doesn't exist yet. That's because the CLI is built _after_ everything is installed
3. The build fails because the CLI is built too late

Splitting out to two separate `npx pnpm` calls resolves the issue, since the `postinstall` script will run before the other workspace dependencies are resolved. 

I don't want to hold up this PR on unrelated tooling issues. So I'll keep the `netlify.toml` as-is, but I think a build pipeline is in our future to orchestrate this nicely. Turborepo perhaps?

Reply:
maybe we can just have an extra Script for this task in package.json, like vitejs does https://github.com/vitejs/vite/blob/main/package.json#L34 

```
ci:www: "run-s build www:build"  
```
and then in [netlify.toml]

```
npx pnpm i --store=node_modules/.pnpm-store && npx pnpm -w run ci:www 
```
what do you think?

Reply:
Update: made an issue! #176